lib/tasks/foreman_ansible_tasks.rake

```diff
@@ -13,19 +13,15 @@ namespace :test do
 end
 
 namespace :foreman_ansible do
-  task :rubocop do
-    begin
-      require 'rubocop/rake_task'
-      RuboCop::RakeTask.new(:rubocop_foreman_ansible) do |task|
-        task.patterns = ["#{ForemanAnsible::Engine.root}/app/**/*.rb",
-                         "#{ForemanAnsible::Engine.root}/lib/**/*.rb",
-                         "#{ForemanAnsible::Engine.root}/test/**/*.rb"]
-      end
-    rescue StandardError
-      puts 'Rubocop not loaded.'
+  begin
+    require 'rubocop/rake_task'
+    RuboCop::RakeTask.new(:rubocop) do |task|
+      task.patterns = ["#{ForemanAnsible::Engine.root}/app/**/*.rb",
+                       "#{ForemanAnsible::Engine.root}/lib/**/*.rb",
+                       "#{ForemanAnsible::Engine.root}/test/**/*.rb"]
```

Comment:
shouldn't this be `Include: ...` in `.rubocop.yml`?

Reply:
It might be worth moving it to `.rubocop.yml` so that we are consistent across plugins...

Reply:
I've tried that, but it seems, that it doesn't work for the rake task actually :disappointed: